benchmarks/benchmark_runner.py

```diff
@@ -155,9 +222,10 @@ def main() -> None:
       model_name=benchmark_model,
       software_config=v6e_env_configs,
       hardware_config=v6e_256_configs,
+      num_steps=options.num_steps,
```

Comment:
num_steps is benchmark specific, can we change it to `xpk_benchmark_runner` optional argment instead of HW config.